src/buffer/cc_buf.c

```diff
@@ -157,7 +157,7 @@ buf_destroy(struct buf **buf)
         return;
     }
 
-    cap = buf_capacity(*buf);
+    cap = buf_size(*buf);
     log_verb("destroy buf %p capacity %"PRIu32, *buf, cap);
```

Comment:
here the capacity is only used for logging, and after the code change the description no longer matches the function called. Maybe rewrite as `log_verb("destroy buf %p size %"PRIu32, *buf, buf_size(*buf));`?

Reply:
the capacity/size here is also used for the stats calculation below, and using `buf_capacity` was giving us incorrect stats. but yes, I will change the log message.